resources/dispute_test.go

```diff
@@ -0,0 +1,73 @@
+package resources_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/razorpay/razorpay-go/constants"
+	"github.com/razorpay/razorpay-go/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+const TestDisputeID = "fake_dispute_id"
+
+func TestDisputeFetch(t *testing.T) {
+	url := "/" + constants.VERSION_V1 + constants.DISPUTE + "/" + TestDisputeID
```

Comment:
```suggestion
	url := fmt.Sprintf("/%s%s/%s", constants.VERSION_V1, constants.DISPUTE, TestDisputeID)
```
lets keep this format consistent

Reply:
updated url format